django-verdant/rca/api/serializers.py

```diff
@@ -149,3 +149,4 @@ class RCAImageSerializer(ImageSerializer):
     thumbnail = ImageRenditionField('max-165x165', read_only=True)
     rca2019_feed_image = ImageRenditionField('fill-878x472', read_only=True)
     rca2019_feed_image_small = ImageRenditionField('fill-80x46', read_only=True)
+    rca2019_supervised_student_image = ImageRenditionField('fill-60x60', read_only=True)
```

Comment:
I think this is overkill  - I can just use the image.thumbnail value which is 160x160